src/store/entity.js

```diff
@@ -112,7 +116,8 @@ export default {
     },
     getPins({ state, commit }) {
       return this.$apis.set.get(state.featuredSetId, {
-        profile: 'itemDescriptions'
+        profile: 'itemDescriptions',
+        pageSize: 100 // TODO: should this limit to 24?
```

Comment:
It makes sense to me to limit this to 24, what's your concern?

Reply:
if you accidentally pinned more items by using multiple tabs for example, then you won't be able to see all of the pinned items. So you can't unpin from the end.

Reply:
I'm going to remove the todo here, because I think there's no harm getting the extra pins _if_ they exist.